.setup/pip/system_requirements.txt

```diff
@@ -21,7 +21,7 @@
 cryptography==3.3.2
 
 # For QR bulk upload
-opencv-python==3.4.9.33
+opencv-python==4.5.5.64
```

Comment:
Per the note above, updating this dependency will not work on Ubuntu 18.04. We need to wait till #7925 is merged.